docs/user/user/adding-glean-to-your-project/qt.md

```diff
@@ -0,0 +1,107 @@
+# Adding Glean to your Qt/QML project
+
+This page provides a step-by-step guide on how to integrate the [Glean.js](https://github.com/mozilla/glean.js/) library into a Qt/QML project.
+
+Nevertheless this is just one of the required steps for integrating Glean successfully into a project. Check you the full [Glean integration checklist](./index.md) for a comprehensive list of all the steps involved in doing so.
+
+## Requirements
+
+* Python >= 3.6.
+
+## Setting up the dependency
+
+Glean.js' Qt/QML build is distributed as an asset with every Glean.js release. In order to download
+the latest version visit [https://github.com/mozilla/glean.js/releases/latest](https://github.com/mozilla/glean.js/releases/latest).
+
+Glean.js is a [QML module](https://doc.qt.io/qt-5/qtqml-modules-topic.html),
+so extract the contents of the downloaded file wherever you keep your other modules.
+Make sure that whichever directory that module is placed in, is part of the
+[QML Import Path](https://doc.qt.io/qt-5/qtqml-syntax-imports.html#qml-import-path).
+
+After doing that, import Glean like so:
+
+```qml
+import org.mozilla.Glean <version>
+```
+
+The `<version>` number is the version of the release you downloaded minus its patch version.
+For example, if you downloaded Glean.js version `0.15.0` your import statement will be:
```

Comment:
We used an [infobox](https://github.com/mozilla/glean/blame/1a33bc846b67ca80de553ba3324646a0b57aca4e/docs/user/user/adding-glean-to-your-project/kotlin.md#L31-L35) for a similar information in the other bindings.